src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,33 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.time.LocalDate;
+
 public class SalaryInfo {
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String report = "Report for period " + dateFrom + " - " + dateTo;
```

Comment:
you missed the point from the checklist
use string builder instead of string concat